components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/BUILD.gn

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("status_tray") {
+  sources = [
+    "brave_vpn_tray_command_ids.h",
+    "brave_vpn_tray_strings_en.h",
+    "status_tray_runner.cc",
+    "status_tray_runner.h",
+  ]
+
+  public_deps = [ "//skia" ]
```

Comment:
why is this a public dep?

Reply:
because skia works only as public

Reply:
discussed in Slack and replaced //skia by //ui/base